content/departments/product-engineering/product/process/rollout_process.md

```diff
@@ -63,11 +61,11 @@ Sourcegraph Cloud is continuously deployed with all new updates to master. We ma
 
 ### After merge, before launch
 
-1. **Gather internal feedback:** Enable the feature flag in the [Sourcegraph organization settings](https://sourcegraph.com/organizations/sourcegraph/settings) to enable your feature for all Sourcegraph team members. Be sure to leave enough time for folks to experience the feature in their workflows and provide feedback.
+1. **Gather internal feedback:** Enable the feature flag in the [Sourcegraph organization settings](https://sourcegraph.com/organizations/sourcegraph/settings) to enable your feature for all Sourcegraph team members. Be sure to leave enough time for folks to experience the feature in their workflows and provide feedback. the #feedback-dogfood channel is a great place to ask for internal feedback.
```

Comment:
Nit: capitalize "the #feedback-dogfood channel is a great..."

Reply:
dammit